h/app.py

```diff
@@ -94,6 +121,13 @@ def includeme(config):
         },
     })
 
+    # After commit events
+    config.add_request_method(EventQueue, name='after_commit_events', reify=True)
```

Comment:
I wonder if we should make the interface for sending an event to be sent after commit look more similar to the normal interface for sending an event? Maybe something like:

```python
request.registry.notify(event)      # Send event immediately.
request.notify_after_commit(event)  # Send event after commit.
```

Just for consistency (maybe it can even be `request.registry.notify_after_commit()`? not sure) but also because I don't think we need the full API of a list object on the request (it looks like code that has access to the request could also do things like delete, insert, reorder the list, not just append to it?)